qa/python_models/bls/model.py

```diff
@@ -275,7 +275,7 @@ def _send_bls_sequence_requests(self, correlation_id, is_decoupled):
             else:
                 infer_response = infer_request.exec()
 
-            self.assertFalse(infer_response.has_error(), infer_response.error())
+            self.assertFalse(infer_response.has_error())
```

Comment:
The correct function should be `infer_response.error().message()` to print out the error string. However, it will throw an error when there is no error object with infer_response if we call `infer_response.error().message()` here, so removing these.